app/core/models/ocelot/package.scala

```diff
@@ -47,6 +52,13 @@ package object ocelot {
   def pageLinkIds(phrases: Seq[Phrase]): List[String] = phrases.flatMap(phrase => pageLinkIds(phrase.english)).toList
   def labelReferences(str: String): List[String] = plSingleGroupCaptures(labelRefRegex, str)
   def labelReference(str: String): Option[String] = plSingleGroupCaptures(labelRefRegex, str).headOption
+  def listLength(listName: String, labels: Labels): Option[String] = labels.valueAsList(listName).fold[Option[String]](None){l => Some(l.length.toString)}
```

Comment:
Should this method really be defined in Labels/LabelCacheImpl?

Reply:
think the Labels interface has all the essential fns using a "minimal and complete" criteria, this is an extra convenience fn used to implement the [list:<label>:length] placeholder when applying the regex.

Reply:
I'll accept that argument.